service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisCoverDoc.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+@Builder(builderClassName = "Builder")
+public class DocmosisCoverDoc {
+    private final String familyManCaseNumber;
+    private final String ccdCaseNumber;
+    private final String representativeName;
+    private final String representativeAddress;
+
+    public Map<String, Object> toMap(ObjectMapper mapper) {
+        return mapper.convertValue(this, new TypeReference<Map<String, Object>>() {});
+    }
+}
```

Comment:
I want this to be abstracted to a class that all these docmosis objects extend. I guess this is fine for now 🤔